src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -208,6 +208,11 @@ public class BuildManager : INodePacketHandler, IBuildComponentHost, IDisposable
         /// </summary>
         private bool _disposed;
 
+        /// <summary>
+        /// When the BuildManager was created.
+        /// </summary>
+        private DateTime _instantiationTimeUtc;
```

Comment:
Does the `Utc` suffix carry information? I thought that was embedded in the `Kind` property.

Reply:
You're right I believe (although if you stray from UTC, it doesn't track anything but "not UTC"). However I am not sure that comparisons of DateTime flag when you are comparing against non-UTC. The concept of Kind was bolted on later. I considered using DateTimeOffset, but the pattern seems to be to use DateTime and the Utc suffix to help visually spot mistakes, eg.,

```c#
        // The newest file what we have seen
        private DateTime _newestFileTimeUtc = DateTime.MinValue;
```
```c#
                    destinationPath.LastWriteTimeUtc = zipArchiveEntry.LastWriteTime.UtcDateTime;
```